### DISCUSSION TITLE
Is podcini android auto enabled ?

### DISCUSSION THREAD

**Discussion by lux70** [XilinJia/Podcini#11] [Q&A]
Is podcini android auto enabled ?
I can wiew the icon , but then black screen appear on the display of the car.

**Comment by XilinJia** [comment]
It should work on a car's android system.  Can you provide more info as to versions of your car's Android and Podcini?

**Reply by lux70** [reply]
I use the latest podcini .. i also use antennapod without problem.
Android 14 ..pixel 7a

**Reply by XilinJia** [reply]
This is the system on your car: Android 14 ..pixel 7a?

**Reply by lux70** [reply]
My phone is pixel 7a with android 14.

My car is toyota corolla hybrid

With antennapod all seems fine. While with podcini after starting the app on the hud of the car, only a black screen appears.  you can't see the interface.. the menus.. nothing.. black

**Reply by XilinJia** [reply]
One thing I suspect is that Android Auto only allows Google approved apps and Podcini is not an approved app by Google, as it's not yet published on Google Play yet.  This will be done shortly.  You can either wait a while, or use some tricks to get around this restriction, which I'm not recommending here.

**Comment by XilinJia** [comment]
Just tested with an Automotive emulator running Android 13 (the newest I can get), Podcini appears to run fine.  Both the free and the play versions.  Podcini 4.2.3

**Comment by XilinJia** [comment]
OK, now 4.6.2 should work with Android Auto.  Please try it and let me know.

**Comment by Watever44** [comment]
I am facing with same issue. Black screen on android auto. 
Using the .R version but the standard do the same thing

Version : 6.3.7 (Unknown commit)
Android Auto: 12.4.642824

Any idea what I can try?
Antenna pod works in debug mode but lacks stuff I like in podcini

**Comment by XilinJia** [comment]
Thanks for reminding.  Will tackle the android auto issue shortly.

**Comment by XilinJia** [comment]
If someone comes here to find out how Podcini can show up on car's HUD with Android Auto, please read AnroidAuto.md for instructions.

**Reply by gonewild8** [reply]
I got the same problem, I can see only a black screen but if I am starting a podcast from the app on the smartphone it will show also on Android Auto. Also the search function on podcini in AA is not working. I have AA developer and unknown sources turned on.

I am on a OnePlus nord 2, android 13, latest version of Podcini from the Play Store


https://github.com/user-attachments/assets/5c6563ba-8045-4b14-b030-16b2af3e9435

![IMG20241213230151](https://github.com/user-attachments/assets/edbc9955-a6dc-430b-9323-96f53136ae75)


https://github.com/user-attachments/assets/bc1ffe4f-6e1d-46d4-abe4-a6dfb6b590c1

**Reply by XilinJia** [reply]
Thanks for reporting. will check.
